test/e2e/tests/hardware-wallets/trezor-sign.spec.ts

```diff
@@ -33,20 +32,18 @@ describe('Trezor Hardware Signatures', function (this: Suite) {
         await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
 
         await driver.clickElement('.confirm-scroll-to-bottom__button');
-        await driver.clickElement({ text: 'Confirm', tag: 'button' });
+        await driver.clickElementAndWaitForWindowToClose({
+          text: 'Confirm',
+          tag: 'button',
+        });
 
-        await driver.waitUntilXWindowHandles(2);
         await driver.switchToWindowWithTitle(WINDOW_TITLES.TestDApp);
         await driver.clickElement('#signTypedDataV4Verify');
 
-        const verifyRecoverAddress = await driver.findElement(
-          '#signTypedDataV4VerifyResult',
-        );
-
-        assert.equal(
-          await verifyRecoverAddress.getText(),
-          KNOWN_PUBLIC_KEY_ADDRESSES[0].address.toLocaleLowerCase(),
-        );
+        await driver.waitForSelector({
+          css: '#signTypedDataV4VerifyResult',
+          text: KNOWN_PUBLIC_KEY_ADDRESSES[0].address.toLocaleLowerCase(),
```

Comment:
minor nit: could we have this in variable and then pass the variable. Its ok if you dont make the change too.
`let address = KNOWN_PUBLIC_KEY_ADDRESSES[0].address.toLocaleLowerCase()`

Reply:
sure I've updated it!

Reply:
I don't think this change is needed here, it's clear enough the origin implementation. 
But since you already made the change, we can keep it.

Reply:
@chloeYue, yeah you're right. After the change was made, I also feel it's not required. I had noted this point..
My initial thought was to read the value, convert it to lowercase and then pass it. 

@seaona, thanks for making the change. Since you've made the adjustment we can skip it for now.